package.json

```diff
@@ -77,6 +78,7 @@
     "standard-version": "^9.3.2",
     "ts-node": "^10.4.0",
     "tsd": "^0.19.1",
+    "typedoc": "^0.22.12",
```

Comment:
I recall Neal mentioning that this didn't play well with npm due to strict TS version requirements, is this no longer the case?

Reply:
I thought that was jsdoc - used to generate the legacy documentation.  @nbbeeken do you remember which package caused problems?

Reply:
It was [typedoc](https://github.com/TypeStrong/typedoc/blob/master/package.json#L30-L32). When the next version of typescript drops we get install errors until typedoc adopts it, which we usually don't need to be blocked on